src/server/scripts/Outland/HellfireCitadel/HellfireRamparts/boss_vazruden_the_herald.cpp

```diff
@@ -349,7 +348,11 @@ class boss_vazruden_the_herald : public CreatureScript
                     if (Creature* Vazruden = me->SummonCreature(NPC_VAZRUDEN, VazrudenMiddle[0], VazrudenMiddle[1], VazrudenMiddle[2], 0, TEMPSUMMON_CORPSE_TIMED_DESPAWN, 6000000))
                         VazrudenGUID = Vazruden->GetGUID();
                     if (Creature* Nazan = me->SummonCreature(NPC_NAZAN, VazrudenMiddle[0], VazrudenMiddle[1], VazrudenMiddle[2], 0, TEMPSUMMON_CORPSE_TIMED_DESPAWN, 6000000))
+                    {
                         NazanGUID = Nazan->GetGUID();
+			            Unit* player = Nazan->SelectNearestPlayer(60.00f);
+			            Nazan->AI()->AttackStart(player);
+		    }
```

Comment:
fix this tabs + spaces mix into spaces.

Reply:
Right, seems I forgot to include this line in my previous comments.
```suggestion
                    }
```